### ISSUE TITLE
Ctrl-C while input() messes up tty

### ISSUE THREAD

**Issue by wwwutz** [wwwutz/Buku#4]
```
wwwutz@eldersoftheinternet$ buku --ai
DB file is being created at /home/wwwutz/.local/share/buku/bookmarks.db.
You should encrypt it.
Add parent folder names as tags? (y/n):  <CTRL-C>
Interrupted.
<ENTER>wwwutz@eldersoftheinternet$ <ENTER>wwwutz@eldersoftheinternet$ wwwutz@eldersoftheinternet$ <ENTER>
<$ stty -a > x.1>
<$ reset>
Erase is backspace.
wwwutz@eldersoftheinternet$ 
wwwutz@eldersoftheinternet$ stty -a > x.2
wwwutz@eldersoftheinternet$ diff x.[12]
4c4
< werase = ^W; lnext = <undef>; flush = ^O; min = 1; time = 0;
---
> werase = ^W; lnext = ^V; flush = ^O; min = 1; time = 0;
6,7c6,7
< -ignbrk -brkint -ignpar -parmrk -inpck -istrip -inlcr -igncr -icrnl ixon -ixoff
< -iuclc -ixany -imaxbel iutf8
---
> -ignbrk brkint ignpar -parmrk -inpck -istrip -inlcr -igncr icrnl ixon -ixoff
> -iuclc -ixany imaxbel iutf8
9c9
< isig -icanon iexten -echo echoe echok -echonl -noflsh -xcase -tostop -echoprt
---
> isig icanon iexten echo echoe echok -echonl -noflsh -xcase -tostop -echoprt
```

**Comment by wwwutz** [comment]
hint: `sigint_handler`

**Comment by wwwutz** [comment]
fixed in #8
